wordle-web/pages/instructions.vue

```diff
@@ -0,0 +1,37 @@
+<template>
```

Comment:
This is good! You might want to add a section that shows what each color means on the game board.